planner/core/logical_plans.go

```diff
@@ -617,3 +619,17 @@ type LogicalLock struct {
 
 	Lock ast.SelectLockType
 }
+
+// LogicalWindow represents a logical window function plan.
+type LogicalWindow struct {
+	logicalSchemaProducer
+
+	WindowFuncDesc *aggregation.WindowFuncDesc
+	ByItems        []property.Item
```

Comment:
1. How about `[]*expression.Column`?
2. It's better to add a comment to specify that `ByItems` is composed of `PARTITION BY` and `ORDER BY` items.

Reply:
No, it is important for columns to have order, for there are queries like `row_numnber() over(order by a desc)`.

Reply:
got it.